srcpkgs/flowblade/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'flowblade'
 pkgname=flowblade
-version=2.4
-revision=2
+version=2.6
+revision=1
 archs=noarch
```

Comment:
please remove noarch as part of the transition away from noarch

Reply:
Of course. Will be changed ASAP. Where can I read more about the transition away from noarch?

Reply:
#24684, #24357